maro/backends/frame.pyx

```diff
@@ -185,20 +186,20 @@ cdef class NodeBase:
         return self._index
 
     cdef void setup(self, BackendAbc backend, int index) except *:
-        """setup frame node, and bind attributes"""
+        """Setup frame node, and bind attributes."""
         self._index = index
         self._backend = backend
 
         self._bind_attributes()
 
     cdef void _bind_attributes(self) except *:
-        """bind attributes declared in class"""
+        """Bind attributes declared in class."""
         cdef dict __dict__ = object.__getattribute__(self, "__dict__")    
 
         cdef str name
         cdef str node_name
         cdef str cb_name
-        cdef NodeAttributeAccessor attr_acc
+        cdef _NodeAttributeAccessor attr_acc
 
         for name, attr in type(self).__dict__.items():
             # append an attribute access wrapper to current instance
```

Comment:
Start with Capital letter, end with period.

Reply:
this is an internal class, here use _ to hide it in api doc.